src/tests/home.test.js

```diff
@@ -0,0 +1,17 @@
+import { seriesCount } from '../components/home';
+
+const getList = jest.fn(async () => [
+  { id: 1, name: 'a', image: { medium: '' } },
+  { id: 2, name: 'a', image: { medium: '' } },
+  { id: 3, name: 'a', image: { medium: '' } },
+]);
+// eslint-disable-next-line no-unused-vars
+const getDetails = jest.fn((id) => getList()
+  .then((list) => list.filter((item) => item.id === id)[0]));
+
+describe('', () => {
+  it('should add a count to the list', async () => {
+    const list = await seriesCount(getList());
```

Comment:
Please ensure that `ALL` the members of the team appear in the video(presentation) and talk about the project according to the project requirement.

Reply:
I'm working alone at the moment.

Reply:
@joshuaivie ,

Please is there any particular reason why you are working alone at the moment?

Reply:
I was unpaired from my partner by student success.

Reply:
@joshuaivie ,

Oh Okay. Kindly make other changes and re-submit for review from your dashboard.